src/selective.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var Crawler = require('./lib/crawler.js').Crawler;
+var _ = require('lodash');
+var src = require('./program');
+
+module.exports = function(ipps, commander) {
+  var logger = commander.quiet ? {log: _.noop, error: _.noop} : console;
+  var maxRequests = commander.max ? parseInt(commander.max, 10) : 100;
+  var crawler = new Crawler(maxRequests, logger);
+  crawler.getSelCrawl(ipps)
+    .then(function(response) {
+      if (commander.store) {
+        src.store(response, commander.store, commander.logsql);
```

Comment:
`dbUrl` is used when we get from the database
`commander.store` is used when we write to the database 

They're different variables. `commander.dbUrl` doesn't exist.
I guess my variable names are somewhat confusing. Sorry about that.

it's all in index.js